packages/react-celo/src/hooks/use-wallet-connect-connector.ts

```diff
@@ -86,6 +96,7 @@ export default function useWalletConnectConnector(
 
         onSubmit(connector);
       } catch (reason) {
+        const CANCELED = Symbol();
```

Comment:
This should probably be the symbol exported from that other place right?

Reply:
Oh oops that part of some testing, good catch!